watershed-fire/1_fetch.R

```diff
@@ -0,0 +1,27 @@
+source("1_fetch/src/get_data.R")
+
+# Fetch input data via download and manual download
+p1_targets <- list(
+  # Download fire perimeter data from MTBS
+  tar_target(perim,
+             get_fire_perim(url = "https://edcintl.cr.usgs.gov/downloads/sciweb1/shared/MTBS_Fire/data/composite_data/burned_area_extent_shapefile/mtbs_perimeter_data.zip",
+                            perim_zip_path = "1_fetch/tmp/mtbs.zip", 
+                            perim_tmp_path = "1_fetch/tmp", 
+                            crs = 9311)),
+  
+  # Load Forests 2 Faucets 2.0 data
+  # Can't figure out how to download directly from Box
+  # Data available from https://usfs-public.app.box.com/v/Forests2Faucets/file/938183618458
+  
+  # To read a geodatabase, file_in = [path and name of geodatabase] and layer = [name of layer]
+  # To read a shapefile, file_in = [path and name shapefile]
+  tar_target(huc,
```

Comment:
here you could use a slightl ymroe informative target name, like `f2f2_huc12` to correspond to the layer you're using